README.md

```diff
@@ -60,13 +73,11 @@ And update your config / options
 
 
 ### passing as an instance
-
 ```
 var S3Adapter = require('parse-server-s3-adapter');
 
-var s3Adapter = new S3Adapter('accessKey',
-                  'secretKey',
-                  'bucket' , {
+var s3Adapter = new S3Adapter(['accessKey',
```

Comment:
i just removed them.  i think that the following section covers the permutations.